waisntechtools/waisntechtools/urls.py

```diff
@@ -13,12 +13,29 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from urllib.parse import urlencode
+
 from django.contrib import admin
 from django.urls import path, include
 from django.views.generic import RedirectView
+from django.conf import settings
+from django.contrib.auth import logout as log_out
+from django.http import HttpResponseRedirect
+
+
+def logout(request):
+    log_out(request)
+    return_to = urlencode({'returnTo': request.build_absolute_uri('/')})
+    logout_url = 'https://%s/v2/logout?client_id=%s&%s' % \
+                 (settings.SOCIAL_AUTH_AUTH0_DOMAIN, settings.SOCIAL_AUTH_AUTH0_KEY, return_to)
+    return HttpResponseRedirect(logout_url)
+
 
 urlpatterns = [
     path('', RedirectView.as_view(url='alerts')),
     path('alerts/', include('alerts.urls')),
     path('admin/', admin.site.urls),
```

Comment:
Should admin login be subject to Auth0?

Reply:
good call!

Reply:
There's no easy way to subject a default url to auth0 authentication. I've removed the url. Since there will be a **/debug** view, the same debugging support can be added there.